hbase_sql/src/main/java/com/splicemachine/olap/OlapServerSubmitter.java

```diff
@@ -244,6 +244,24 @@ public void run() {
 
     }
 
+    private byte[] getKeepAlivePayload() {
+        DatabaseVersion version;
+        try {
+            version = JMXUtils.getLocalMBeanProxy(JMXUtils.SPLICEMACHINE_VERSION, DatabaseVersion.class);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Is it needed to wrap the exception in a `RuntimeException`, can we just remove the catch?

Reply:
I removed that as part of the refactoring to address other comments